docs/user_guide.rst

```diff
@@ -40,11 +40,6 @@ To use these:
 
 Customising plots
 -----------------
-`Matplotlib style sheets <https://matplotlib.org/stable/tutorials/introductory/customizing.html#defining-your-own-style>`__ can be used to customise
-the plots generated by ``napari-matplotlib``.
-To use a custom style sheet:
-
-1. Save it as ``napari-matplotlib.mplstyle``
-2. Put it in the Matplotlib configuration directory.
-   The location of this directory varies on different computers,
-   and can be found by calling :func:`matplotlib.get_configdir()`.
```

Comment:
👍 
Does this warrant a major version bump?

Reply:
Yes, hence the 2.0 milestone 😉